keyboards/rk/rk750/rk750.c

```diff
@@ -0,0 +1,10 @@
+// Copyright 2024 yangzheng20003 (@yangzheng20003)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "quantum.h"
+#define LED_ENABLE_PIN A5
+void keyboard_pre_init_kb(void) {
+    gpio_set_pin_output(LED_ENABLE_PIN);
+    gpio_write_pin_high(LED_ENABLE_PIN);
```

Comment:
What exactly is this for? All the results I'm seeing for this keyboard on Google indicate that this is a wireless keyboard. If this is initialisation code for a wireless module, this PR will not be accepted without the accompanying wireless code.